accounts/models.py

```diff
@@ -116,6 +116,15 @@ def get_complete_address(self):
             return f'{self.street_address1}, {self.street_address2}, {self.city} - {self.pincode}, {self.state}'
         return f'{self.street_address1}, {self.city} - {self.pincode}, {self.state}'
 
+    @property
+    def get_profile_image_url(self):
+        if self.profile_image and hasattr(self.profile_image, 'url'):
+            return self.profile_image.url
+        elif self.gender == 'F':
+            return "/static/home/images/woman.png"
```

Comment:
Instead of the absolute address to static, it would be better to use `STATIC_URL` from settings somehow and then append the rest of the address after that. 
It is necessary because it may happen that we use some other server or AWS bucket for hosting static files sometime in future.

Reply:
Okay, I am trying to do this and also Thanks to draw my attention to production environment for static files.

Reply:
> Instead of the absolute address to static, it would be better to use `STATIC_URL` from settings somehow and then append the rest of the address after that.
> It is necessary because it may happen that we use some other server or AWS bucket for hosting static files sometime in future.

[`Done`](https://github.com/garg3133/JagratiWebApp/pull/46/commits/ba1009dace57c45d377aab3b16b9d71a04554c5d#diff-6b070a300f0e440e2b6e3ae75cd4a54300517cbc9bdbb9a08584f10c026abaa1L124-R126)

Reply:
You may click on `Resolve Conversation` after you're done with any review.

Reply:
Ok 👍🏻